test/rendering/cases/webgl-vectortile-masking/main.js

```diff
@@ -46,7 +46,7 @@ class WebGLVectorTileLayer extends VectorTile {
       className: this.getClassName(),
       style: {
         'fill-color': '#eee',
-        'stroke-color': 'rgb(136,136,136, 0.5)',
+        'stroke-color': 'rgba(136,136,136, 0.5)',
```

Comment:
This looks like a typo in the rendering test.  The previous parser parsed this as an `rgba()` color.  The new parser ignores the fourth value in the `rgb()` list.  Changing the test to use `rgba()` matches the expected output (where the stroke has an alpha value applied).

Reply:
Thanks, good catch!

Reply:
There was one other case that I updated as well.  Rendering tests are passing now.  Let me know if you approve.  I'd like to base some color interpolation work on this.